lib/loglevel.js

```diff
@@ -99,57 +99,63 @@
         storageKey = undefined;
       }
 
+      /** 
+       * The categories is the inherited hierarchy of this logger, used
+       * to get parent categories/definitions.
+       */
+      this.categories = categoriesParam && categoriesParam.slice();
+      
       function persistLevelIfPossible(levelNum) {
           var levelName = (logMethods[levelNum] || 'silent').toUpperCase();
 
           if (typeof window === undefinedType || !storageKey) return;
 
           // Use localStorage if available
           try {
-              window.localStorage[storageKey] = levelName;
-              return;
-          } catch (ignore) {}
-
-          // Use session cookie as fallback
-          try {
-              window.document.cookie =
-                encodeURIComponent(storageKey) + "=" + levelName + ";";
-          } catch (ignore) {}
-      }
-
-      function getPersistedLevel() {
-          var storedLevel;
-
-          if (typeof window === undefinedType || !storageKey) return;
-
-          try {
-              storedLevel = window.localStorage[storageKey];
-          } catch (ignore) {}
+                window.localStorage[storageKey] = levelName;
+                return;
+            } catch (ignore) {}
+
+            // Use session cookie as fallback
+            try {
+                window.document.cookie =
+                    encodeURIComponent(storageKey) + "=" + levelName + ";";
+            } catch (ignore) {}
+        }
```

Comment:
I think the indentation here is messed up? It seems like this is also causing unnecessary indentation changes to all the code after this, making the diff a little harder to follow (lots of things getting marked as changed that are not, and getting interleaved in the diff with new code when it otherwise would not be).

Reply:
According to the editor config, it is supposed to be 4 spaces indent, which is what this is.  It was previously a mix of 4 and 2 spaces.  I'm fine/better off with 2, just saying that the configuration says 4.   Note that is NOT true for JSON, which IS 2 spaces.

Reply:
Right. I was referring to the fact that ([at the time of this comment](https://github.com/pimterry/loglevel/blob/919402bf66132be5b32fd035e78351452d74af35/lib/loglevel.js)), line 115 was indented 6 from line 114 instead 4, and then when dedenting on line 117 and later, everything was off by 2 (outdented 4 instead of 6). But it looks like you fixed that in 3b00cad2688ffc667dea164b5108bbcb3128ec37 🙂